File: repos_estudiantes_ALVAREZ_primera_entrega/CARDON_PAZ_LUIS_ABDEL/POLIMORFISMO/EJERCICIO_7/Parada.java
package POLIMORFISMO.EJERCICIO_7;

public class Parada {
    private int nroAdmins;
    private int nroAutos;
    private String[] admins;
    private String[][] autos;
    
    public Parada() {
        this.nroAdmins = 0;
        this.nroAutos = 0;
        this.admins = new String[10];
        this.autos = new String[10][3];
    }
    
    public Parada(int nroAdmins, int nroAutos) {
        this.nroAdmins = 0;
        this.nroAutos = 0; 
        this.admins = new String[10];
        this.autos = new String[10][3];
    }
    
    public void adicionar(String admin) {
        if (nroAdmins < 10) {
            admins[nroAdmins] = admin;
            nroAdmins++;
        }
    }
    
    public void adicionar(String conductor, String placa, String nombreP) {
        if (nroAutos < 10) {
            autos[nroAutos][0] = conductor;
            autos[nroAutos][1] = placa;
            autos[nroAutos][2] = nombreP;
            nroAutos++;
        }
    }
    
    public void mostrar() {
        System.out.println("Número de Admins: " + nroAdmins);
        System.out.println("Número de Autos: " + nroAutos);
        
        System.out.println("Administradores:");
        for (int i = 0; i < nroAdmins; i++) {
            System.out.println("- " + admins[i]);
        }
        
        System.out.println("Autos:");
        for (int i = 0; i < nroAutos; i++) {
            System.out.println("- Conductor: " + autos[i][0] + ", Placa: " + autos[i][1] + ", NombreP: " + autos[i][2]);
        }
    }
}

File: repos_estudiantes_TARQUI_primera_entrega/REQUENA_OROS_ADRIAN_MARCELO/Polimorfismo/Polimorfismo/ejercicio7/Parada.cs
using System;

namespace ejercicio7
{
	public class Parada
	{
	    private string[] admins = new string[10];
	    private string[,] autos = new string[10, 3];
	    private string nombreP;
	    private int nroAdmins = 0;
	    private int nroAutos = 0;
	
	    // Constructores
	    public Parada()
	    {
	        nombreP = "Parada sin nombre";
	    }
	
	    public Parada(string nombre)
	    {
	        nombreP = nombre;
	    }
	
	    // Método para mostrar información
	    public void Mostrar()
	    {
	        Console.WriteLine("\n=== INFORMACIÓN COMPLETA DE LA PARADA ===");
	        Console.WriteLine("Nombre: " + nombreP);
	        Console.WriteLine("\nAdministradores registrados (" + nroAdmins + "):");
	        for (int i = 0; i < nroAdmins; i++)
	        {
	            Console.WriteLine("[" + (i+1) + "] " + admins[i]);
	        }
	        
	        Console.WriteLine("\nVehículos registrados (" + nroAutos + "):");
	        for (int i = 0; i < nroAutos; i++)
	        {
	            Console.WriteLine("Auto #" + (i+1));
	            Console.WriteLine("  Modelo  : " + autos[i,0]);
	            Console.WriteLine("  Conductor: " + autos[i,1]);
	            Console.WriteLine("  Placa    : " + autos[i,2]);
	            Console.WriteLine("----------------------");
	        }
	    }
	
	    // Métodos para agregar elementos
	    public void AdicionarAdmin(string nombreAdmin)
	    {
	        if (nroAdmins < 10)
	        {
	            admins[nroAdmins] = nombreAdmin;
	            nroAdmins++;
	            Console.WriteLine("Administrador agregado: " + nombreAdmin);
	        }
	        else
	        {
	            Console.WriteLine("¡Alerta! Capacidad máxima de administradores alcanzada");
	        }
	    }
	
	    public void AdicionarAuto(string modelo, string conductor, string placa)
	    {
	        if (nroAutos < 10)
	        {
	            autos[nroAutos, 0] = modelo;
	            autos[nroAutos, 1] = conductor;
	            autos[nroAutos, 2] = placa;
	            nroAutos++;
	            Console.WriteLine("Auto registrado: " + modelo + " - " + placa);
	        }
	        else
	        {
	            Console.WriteLine("¡Alerta! No hay espacio para más vehículos");
	        }
	    }
	}
}
